package.json

```diff
@@ -21,6 +21,7 @@
     "gatsby-source-filesystem": "^2.3.34",
     "gatsby-source-mongodb": "^2.3.12",
     "gatsby-transformer-sharp": "^2.5.17",
+    "mocha": "^8.1.3",
```

Comment:
Is this a necessary dependency to run cypress?

Reply:
No,I think that is not necessary. Thank you, I will remove that.